src/ctia/bulk/schemas.clj

```diff
@@ -28,20 +28,23 @@
   "Error related to one entity of the bulk"
   {:error s/Any})
 
-; TODO def => defn
-(s/defschema Bulk
-  (entities-bulk-schema (entities/all-entities) :schema))
+(s/defn get-entities :- [s/Any]
+  "Returns list of enabled entities"
+  [{{:keys [enabled?]} :FeaturesService} :- APIHandlerServices]
+  (->> (entities/all-entities) (filter (fn [[k _]] (enabled? k)))))
 
-; TODO def => defn
-(s/defschema StoredBulk
-  (entities-bulk-schema (entities/all-entities) :stored-schema))
+(s/defn Bulk :- s/Any
```

Comment:
This might be a more informative return schema https://github.com/threatgrid/ctia/blob/e6b45249024c73093ae7692e9b7ff1cbef207f7d/src/ctia/schemas/core.clj#L101

Ditto the other `defn`s in this ns.